redis/5.3/modules/ROOT/pages/redis-connector-reference.adoc

```diff
@@ -1,12 +1,11 @@
 = Redis Connector 5.3 Reference - Mule 4
-:page-aliases: connectors::redis/redis-connector-reference.adoc
 
 Support Version: Select
 
 Release Notes: xref:release-notes::connector/redis-connector-release-notes-mule-4.adoc[Redis Connector Release Notes]
 
 [NOTE]
-You can set configurations in Expression mode, except when Redis Connector is used in conjunction with Object Store Connector, which does not support Expression mode. 
+You can set configurations in Expression mode, except when Redis Connector is used in conjunction with Object Store Connector, which does not support Expression mode.
```

Comment:
Not sure of the change here? Removed a space? Any way to change: 'is used'?

Reply:
No changes here